src/rdb.h

```diff
@@ -95,10 +95,11 @@
 #define RDB_TYPE_ZSET_LISTPACK 17
 #define RDB_TYPE_LIST_QUICKLIST_2   18
 #define RDB_TYPE_STREAM_LISTPACKS_2 19
+#define RDB_TYPE_LIST_LISTPACK 20
```

Comment:
i know... planning to review it too soon.
but still i think there's no reason to change the format for lists, and we can easily fake a one node quicklist instead.

Reply:
Yeah, it sounds OK.

Reply:
I removed the code related to QUICKLIST_CONVERT_THRESHOLD, mainly for 2 reasons:
1. because of the threshold, if we save the listpack as a fake quicklist, we will not be able to determine the encoding before it is saved when rdb loading.
This would go against the fact that we should keep a key in the same encoding when reloading.
2. After a simple benchmarking, I found that the conversion overhead between listpack <-> quicklist is negligible.